demo/data/rags_compare.csv

```diff
@@ -0,0 +1,10 @@
+question
```

Comment:
I'm confused that are we using huggingface_reader ? or we still need this csv file

Reply:
we are using huggingface_reader. if we need using csv file, we can change it in yaml file and pass the file name to 'r_m_ps' in rags_compare.py's line:352, [to "r_m_ps=retriever_model_prompt(filename)"]